.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Checklist
+
+* [ ] I have updated the end-to-end tests to work with my changes
+* [ ] I have updated the OpenAPI spec
+* [ ] I have added styling for Dark Mode
+* [ ] I have added translations for new timeline events
```

Comment:
I think this is also an `opg-sirius` thing since it affects the old frontend?

Reply:
Ah yes you're right, i've got muddled up on this :confused: